TMDbLib/Objects/TvShows/TvShow.cs

```diff
@@ -71,9 +71,15 @@ public class TvShow
         [JsonProperty("last_air_date")]
         public DateTime? LastAirDate { get; set; }
 
+        [JsonProperty("last_episode_to_air")]
+        public TvEpisode LastEpisodeToAir { get; set; }
+
         [JsonProperty("name")]
         public string Name { get; set; }
 
+        [JsonProperty("next_episode_to_air")]
+        public TvEpisode NextEpisodeToAir { get; set; }
```

Comment:
Are we sure of the type? .. [The docs](https://developers.themoviedb.org/3/tv/get-tv-details) only specify `null`.

Reply:
From testing it seems to be the exact same type as `last_episode_to_air`.

https://api.themoviedb.org/3/tv/1412?api_key=%APIKEY%&language=en-US